cmake/modules/FindGMP.cmake

```diff
@@ -0,0 +1,139 @@
+# Try to find the GMP library
```

Comment:
This file is essentially the same with FindMPFR.cmake.

Reply:
oops committed accidentally ... meant to convert to proper FindGMP, since only GMPXX is needed by the compiler, MPFR is only needed for high-precision testing of the library

Reply:
A disheartening complication is that Windows never has GMP. But the Windows MPFR project provides duplicate headers gmpxx.h . That's why my first crack at cmake+cmake build simply requires MPFR.